pom.xml

```diff
@@ -222,9 +223,9 @@
       <profile>
           <id>jboss-repository</id>
           <repositories>
-              <repository>
-                  <id>jboss</id>
-                  <url>http://repository.jboss.org/nexus/content/groups/public/</url>
+              <repository>
+                  <id>jboss</id>
+                  <url>http://repository.jboss.org/nexus/content/groups/public/</url>

```

Comment:
I have no idea why Github shows these lines as modified.  I did not touch it and my local git log does not show any change in this line.